packages/library/src/components/bal-select-option/bal-select-option.tsx

```diff
@@ -14,25 +18,27 @@ export class SelectOption implements ComponentInterface {
 
   @Element() el!: HTMLElement
 
-  /**
-   * The value of the dropdown item. This value will be returned by the parent `<bal-dropdown>` element.
-   */
-  @Prop({ reflect: true }) value: string | undefined
+  @State() isConnected = false
 
   /**
    * Label will be shown in the input element when it got selected
    */
   @Prop() label: string | undefined
 
   /**
-   * If `true` the option is hidden
+   * If `true` the option has a checkbox
    */
-  @Prop({ reflect: true }) hidden = true
+  @Prop() checkbox = false
 
   /**
-   * Baloise icon as a prefix
+   * The value of the dropdown item. This value will be returned by the parent `<bal-dropdown>` element.
```

Comment:
Strange as we have here a `bal-select-option` => `bal-select` and here you talk about `<bal-dropdown>`

Reply:
done